lib/Components/Home.dart

```diff
@@ -69,6 +70,12 @@ class _MyHomePageState extends State<MyHomePage> {
                 actionFunction: () => print("Wow")
               ),
             ),
+            Padding(
```

Comment:
Rather setting a margin-left of 35px I would instead recommend giving the padding widget a centre widget inside of it with the divider inside of that. So Padding(Top, Bottom only) -> Centre -> DoubleDivider.
I would change the top and bottom margins to be 25px

Reply:
@JamieC1998 the bottom padding at 25 doesn't do it on the SGII, only does it properly... 
![bottom padding at 0](https://user-images.githubusercontent.com/20342200/68917206-2dcf3d80-0761-11ea-8015-a31fd6acb213.png)

![bottom padding at 25](https://user-images.githubusercontent.com/20342200/68917215-345db500-0761-11ea-9cd3-356ba32c452b.png)


as well, when turned the phone I got this: 
![horizontal issue](https://user-images.githubusercontent.com/20342200/68917252-48a1b200-0761-11ea-8edf-a9b89dc624b6.png)